Part2_Mapplots/ex2_2_psyplot.ipynb

```diff
@@ -193,9 +193,10 @@
     "ax = plt.axes(projection=ccrs.Robinson())\n",
```

Comment:
<p>A few more words about iconarray and what it does.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/C2SM/pyvis/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>I added more words about the formatoptions and iconarray. Is it better understandable now?</p>

Reply:
Yes, thanks